test/__mocks__/overseas.entity.due.diligence.mock.ts

```diff
@@ -7,10 +7,10 @@ import {
 } from "./fields/address.mock";
 import { DATE, EMPTY_DATE, IDENTITY_DATE_REQ_BODY_MOCK } from "./fields/date.mock";
 
-const TEN_CHARACTERS_LENGTH = "LKJHG.asdf";
+const SIX_CHARACTERS_LENGTH = "LKJHG.";
```

Comment:
Sorry, should have mentioned this earlier, but rather than duplicate these constants twice, suggest adding/reusing the ones already there in `validation.mock.ts`.